Pipfile

```diff
@@ -8,6 +8,8 @@ django = "*"
 
 [dev-packages]
 flake8 = "*"
+pytest = "*"
```

Comment:
Why do you need the `pytest` package? Isn't `pytest-django` enough?

Reply:
I followed Edward installing guide, from my understanding, `pytest-django` is a plugin for `pytest`.